src/magic_folder/util/eliotutil.py

```diff
@@ -117,31 +123,101 @@ def validator(v):
     return validator
 
 
+def opt_eliot_fd(self, fd):
```

Comment:
Yes, I did see it. I had considered using it, but decided not to. I don't recall the exact reason, but I suspect it was partly because I was unsure about the file syntax it uses, and didn't want to spend to much time thinking about it.

Reply:
Okay.  Please file a ticket for unifying these two related things.

Reply:
#466